sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/ExperimentStore.java

```diff
@@ -1,22 +1,19 @@
 /*
- * Copyright (c) 2012. The Genome Analysis Centre, Norwich, UK
- * MISO project contacts: Robert Davey @ TGAC
- * *********************************************************************
+ * Copyright (c) 2012. The Genome Analysis Centre, Norwich, UK MISO project contacts: Robert Davey @
```

Comment:
Are we removing these headers as we find them?

Reply:
Yes, it's been removed now